tools/serverpod_cli/lib/src/create/database_setup.dart

```diff
@@ -6,9 +6,18 @@ import 'package:serverpod_cli/src/logger/logger.dart';
 import 'package:serverpod_cli/src/util/command_line_tools.dart';
 
 class DatabaseSetup {
-  static Future<bool> createDefaultMigration(Directory dir, String name) async {
+  static Future<bool> createDefaultMigration(
+    Directory dir,
+    String name,
+  ) async {
     log.debug('Creating default migration.');
 
+    var config = await GeneratorConfig.load(dir.path);
```

Comment:
Question: What config is loaded here? The one in the newly created project?

Reply:
yes